### PR TITLE
Fix inconsistent spellings of JavaScript 

### PR DESCRIPTION
**Pull request by KaranKaira**
## Description
Edited incorrect spelling of JavaScript(Javascript) 
image

## Related Issue
Fixes issue #5642

### CODE REVIEW

**Comment by joeurassa** [comment]
add me pls

File: src/content/translations/cs/javascript/index.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ---
-title: Ethereum pro vývojáře v JavaScriptu
-description: Naučte se vyvíjet pro Ethereum pomocí projektů a nástrojů založených na JavaScriptu
+title: Ethereum pro vývojáře v JavaScript
```
Comment (samajammin) [inline src/content/translations/cs/javascript/index.md:2]: I suspect we may want to leave translations alone - these may be intentional changes. What do you think @lukassim @minimalsm?

Comment (minimalsm) [reply src/content/translations/cs/javascript/index.md:2]:
```suggestion
title: Ethereum pro vývojáře v JavaScriptu
```

File: src/content/translations/ar/javascript/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ lang: ar
 sidebar: true
 ---
 
-# إثيريوم لمطوري JavaScript {#ethereum-for-javascript-devs}
+# إثيريوم لمطوري JavaScript {#ethereum-for-JavaScript-devs}
```
Comment (minimalsm) [inline src/content/translations/ar/javascript/index.md:8]:
```suggestion
# إثيريوم لمطوري JavaScript {#ethereum-for-javascript-devs}
```

File: src/content/translations/cs/javascript/index.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+description: Naučte se vyvíjet pro Ethereum pomocí projektů a nástrojů založených na JavaScript
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:3]:
```suggestion
description: Naučte se vyvíjet pro Ethereum pomocí projektů a nástrojů založených na JavaScriptu
```

File: src/content/translations/cs/javascript/index.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 lang: cs
 sidebar: true
 sidebarDepth: 1
 ---
 
-# Ethereum pro vývojáře v JavaScriptu {#ethereum-for-javascript-devs}
+# Ethereum pro vývojáře v JavaScript {#ethereum-for-javascript-devs}
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:9]:
```suggestion
# Ethereum pro vývojáře v JavaScriptu {#ethereum-for-javascript-devs}
```

File: src/content/translations/cs/javascript/index.md [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
 
-<div class="featured">Naučte se vyvíjet pro Ethereum pomocí projektů a nástrojů založených na JavaScriptu</div>
+<div class="featured">Naučte se vyvíjet pro Ethereum pomocí projektů a nástrojů založených na JavaScript</div>
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:11]:
```suggestion
<div class="featured">Naučte se vyvíjet pro Ethereum pomocí projektů a nástrojů založených na JavaScriptu</div>
```

File: src/content/translations/cs/javascript/index.md [hunk 6]
```diff
@@ -1,20 +1,20 @@
...
 
 Na platformě Ethereum můžete vytvářet decentralizované aplikace (neboli dapps), které využívají výhody kryptoměn a blockchainové technologie. Tyto aplikace mohou být důvěryhodné, což znamená, že jakmile je jednou nasadíte na Ethereum, budou vždy spouštěny přesně tak, jak jsou naprogramovány. Aplikace dapp mohou kontrolovat digitální aktiva, a tím vytvářet nové druhy finančních aplikací. Mohou být decentralizované, což znamená, že je nemůže ovládat jediná entita nebo osoba a že jsou téměř necenzurovatelné.
 
 ## Začínáme s chytrými kontrakty a jazykem Solidity {#getting-started-with-smart-contracts-and-solidity}
 
-**Udělejte své první kroky k integraci JavaScriptu s Ethereem**
+**Udělejte své první kroky k integraci JavaScript s Ethereem**
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:17]:
```suggestion
**Udělejte své první kroky k integraci JavaScriptu s Ethereem**
```

File: src/content/translations/cs/javascript/index.md [hunk 7]
```diff
@@ -25,9 +25,9 @@ Potřebujete nejdříve úplně základní informace? Podívejte se na [ethereum
 
 - [Full Stack Hello World Voting Ethereum Dapp Tutorial](https://medium.com/@mvmurthy/full-stack-hello-world-voting-ethereum-dapp-tutorial-part-1-40d2d0d807c2)
 
-## Projekty a nástroje v JavaScriptu {#javascript-project-and-tools}
+## Projekty a nástroje v JavaScript {#javascript-project-and-tools}
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:28]:
```suggestion
## Projekty a nástroje v JavaScriptu {#javascript-project-and-tools}
```

File: src/content/translations/cs/javascript/index.md [hunk 8]
```diff
@@ -38,27 +38,27 @@ Potřebujete nejdříve úplně základní informace? Podívejte se na [ethereum
...
-**Ethereum Grid -** **desktopová aplikace s nástroji pro Ethereum, rozšiřitelná pomocí JavaScriptu.**
+**Ethereum Grid -** **desktopová aplikace s nástroji pro Ethereum, rozšiřitelná pomocí JavaScript.**
 
 - [grid.ethereum.org](https://grid.ethereum.org)
 - [GitHub](https://github.com/ethereum/grid)
 
 Hledáte další informační zdroje? Zkuste [ethereum.org/developers.](/developers/)
 
-## Přispěvatelé do komunity v JavaScriptu {#javascript-community-contributors}
+## Přispěvatelé do komunity v JavaScript {#javascript-community-contributors}
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:62]:
```suggestion
## Přispěvatelé do komunity v JavaScriptu {#javascript-community-contributors}
```

File: src/content/translations/cs/javascript/index.md [hunk 9]
```diff
@@ -38,27 +38,27 @@ Potřebujete nejdříve úplně základní informace? Podívejte se na [ethereum
...
 - [Dokumentace](https://docs.ethers.io/)
 - [GitHub](https://github.com/ethers-io/ethers.js/)
 
-**ethereumjs-vm -** **_Ethereum virtuální stroj (VM) implementovaný v JavaScriptu_**
+**ethereumjs-vm -** **\_Ethereum virtuální stroj (VM) implementovaný v JavaScript**
 
 - [GitHub](https://github.com/ethereumjs/ethereumjs-vm)
 
-**Ethereum Grid -** **desktopová aplikace s nástroji pro Ethereum, rozšiřitelná pomocí JavaScriptu.**
+**Ethereum Grid -** **desktopová aplikace s nástroji pro Ethereum, rozšiřitelná pomocí JavaScript.**
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:55]:
```suggestion
**Ethereum Grid -** **desktopová aplikace s nástroji pro Ethereum, rozšiřitelná pomocí JavaScriptu.**
```

File: src/content/translations/cs/javascript/index.md [hunk 10]
```diff
@@ -38,27 +38,27 @@ Potřebujete nejdříve úplně základní informace? Podívejte se na [ethereum
...
 - [GitHub](https://github.com/ethereum/web3.js/)
 
-**Ethers.js -** **_kompletní implementace peněženky pro Ethereum a nástrojů v JavaScriptu a v TypeScriptu_**
+**Ethers.js -** **_kompletní implementace peněženky pro Ethereum a nástrojů v JavaScript a v TypeScriptu_**
 
 - [Dokumentace](https://docs.ethers.io/)
 - [GitHub](https://github.com/ethers-io/ethers.js/)
 
-**ethereumjs-vm -** **_Ethereum virtuální stroj (VM) implementovaný v JavaScriptu_**
+**ethereumjs-vm -** **\_Ethereum virtuální stroj (VM) implementovaný v JavaScript**
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:51]:
```suggestion
**ethereumjs-vm -** **\_Ethereum virtuální stroj (VM) implementovaný v JavaScriptu**
```

File: src/content/translations/cs/javascript/index.md [hunk 11]
```diff
@@ -38,27 +38,27 @@ Potřebujete nejdříve úplně základní informace? Podívejte se na [ethereum
...
 - [GitHub](https://github.com/trufflesuite/truffle)
 
-**Web3.js -** **_javascriptové API pro Ethereum._**
+**Web3.js -** **_JavaScript API pro Ethereum._**
 
 - [Dokumentace](https://web3js.readthedocs.io/en/1.0/)
 - [GitHub](https://github.com/ethereum/web3.js/)
 
-**Ethers.js -** **_kompletní implementace peněženky pro Ethereum a nástrojů v JavaScriptu a v TypeScriptu_**
+**Ethers.js -** **_kompletní implementace peněženky pro Ethereum a nástrojů v JavaScript a v TypeScriptu_**
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:46]:
```suggestion
**Ethers.js -** **_kompletní implementace peněženky pro Ethereum a nástrojů v JavaScriptu a v TypeScriptu_**
```

File: src/content/translations/cs/javascript/index.md [hunk 12]
```diff
@@ -25,9 +25,9 @@ Potřebujete nejdříve úplně základní informace? Podívejte se na [ethereum
 
 - [Full Stack Hello World Voting Ethereum Dapp Tutorial](https://medium.com/@mvmurthy/full-stack-hello-world-voting-ethereum-dapp-tutorial-part-1-40d2d0d807c2)
 
-## Projekty a nástroje v JavaScriptu {#javascript-project-and-tools}
+## Projekty a nástroje v JavaScript {#javascript-project-and-tools}
 
-**Solidity -** **_Nejoblíbenější jazyk pro Ethereum, inspirovaný JavaScriptem._**
+**Solidity -** **_Nejoblíbenější jazyk pro Ethereum, inspirovaný JavaScript._**
```
Comment (minimalsm) [inline src/content/translations/cs/javascript/index.md:30]:
```suggestion
**Solidity -** **_Nejoblíbenější jazyk pro Ethereum, inspirovaný JavaScriptem._**
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

**Review by minimalsm — APPROVED** [review]
Thanks @KaranKaira 

Just made a small tweak to fix some non-English languages that lost some context because of this. All done now and merging 🎉

File: src/content/translations/cs/javascript/index.md (hunk 1 above)
Comment (lukassim) [reply src/content/translations/cs/javascript/index.md]: Agreed, most languages have case endings to indicate a grammatical case, so we should be careful to retain these.

**Comment by KaranKaira** [comment]
Thanks @minimalsm a lot. This is my first contribution ever in Open Source and yes I would like to keep contributing.

**Pull request merged**